package.json

```diff
@@ -39,7 +39,9 @@
     "test:ci": "REACT_APP_ENV=test yarn test --ci --coverage --json --watchAll=false --testLocationInResults --runInBand --outputFile=jest.results.json",
     "update-mocks": "./scripts/update-mocks.sh",
     "sentry-upload-sourcemaps": "sentry-cli --auth-token $SENTRY_AUTH_TOKEN releases -o $SENTRY_ORG -p $SENTRY_PROJECT files $npm_package_version upload-sourcemaps ./build/static/js/",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "cypress:open": "cypress open",
+    "cypress:run": "cypress run"
```

Comment:
Do I get it right that it will run on `safe-team.dev.gnosisdev.com`?
How do I launch it on localhost?

Reply:
you can change baseUrl in cypress.json to localhost:3000/app

Reply:
I've changed the `baseUrl` to test against **localhost**